src/reset.scss

```diff
@@ -0,0 +1,48 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
I'm not sure the conventions for this - but I haven't seen CSS and JS in the same directory before. Might be worth checking if it's worth splitting them out! 👍